plugins/postgres/dbt/include/postgres/macros/catalog.sql

```diff
@@ -28,8 +28,11 @@
     join pg_catalog.pg_class tbl on tbl.relnamespace = sch.oid
     join pg_catalog.pg_attribute col on col.attrelid = tbl.oid
 
-    where sch.nspname != 'information_schema'
-      and sch.nspname not like 'pg\_%' -- avoid postgres system schemas, '_' is a wildcard so escape it
+    where (
+        {%- for schema in schemas -%}
+          sch.nspname = '{{ schema }}'{%- if not loop.last %} or {% endif -%}
```

Comment:
should these predicates be case-insensitive on pg/redshift? I think that would mirror the existing behavior